.gitignore

```diff
@@ -1,16 +1,11 @@
 bin
 debug
-dep
-obj
-src/glyphs.c
-src/glyphs.h
+build
 copy.sh
 *.pyc
-tests/elfs/
 tests/functional/snapshots-tmp/
 tests/unit-tests/qatozil/*.o
 tests/unit-tests/qatozil/*.o.d
 tests/unit-tests/qatozil/qatozil
 
 scan-build/
-tests/elfs/
```

Comment:
I believe that the `build-all.sh` still makes use of `tests/elfs`. See https://github.com/LedgerHQ/app-zilliqa/blob/develop/tools/build-all.sh

Reply:
build all script updated!

Reply:
Thanks!